docs/getting-started/zns-emulation.md

```diff
@@ -39,7 +51,9 @@ a sparse file or the *dd* command to create a fully allocated file.
 -rw-r--r-- 1 root root 34359738368 Jun 21 15:13 /var/lib/qemu/images/zns.raw
 ```
 
-Or using dd:
+##### Using dd to create an Emulated Zone Namespace Backstore
```

Comment:
I'll suggest only describe one way to do it. I prefer to show the truncate version, as that doesn't preallocate the space.

Reply:
I've rewritten this section so that only "truncate" is mentioned as a way to create a backstore.